Jenkinsfile.download_logs

```diff
@@ -3,6 +3,9 @@ pipeline {
 
     parameters {
         string(name: 'REMOTE_SERVICE_URL', defaultValue: 'https://api.openshift.com', description: 'Service URL')
+        string(name: 'DOWNLOAD_ALL', defaultValue: '--download-all', description: 'Collect all logs (not only from failed clusters)')
```

Comment:
Change both `DOWNLOAD_ALL` and `MUST_GATHER` to boolean parameters.
If this parameter was ticked - add the matching flag.

Default `DOWNLOAD_ALL` is `false`, while `MUST_GATHER` is `true`